draft-decraeneginsberg-lsr-isis-fast-flooding.xml

```diff
@@ -861,18 +944,31 @@
       consensus. Future revisions of this document may include new/altered
       text relevant to these issues.</t>
 
-      <t>Issue 1: Placement of the discussion of LPP <xref target="LPP"/>may
-      become congestion control algorithm specific.</t>
-
-      <t>Issue 2: Further discussion of the relevance of the router
-      architecture to feasibility of different flow/congeston control
-      algorithms.</t>
-
-      <t>Issue 3: Discussion of LAN belongs in a separate section - not
-      bundled w the new TLV description.</t>
-
-      <t>Issue 4: Description of new TLV should not limit values advertised
-      to receive only.</t>
+      <t>
+	Issue 1: Placement of the discussion of LPP <xref
+	target="LPP"/> may become congestion control algorithm
+	specific. This has been addressed. Do we agree that this is
+	resolved?
```

Comment:
I agree that all issues can be considered resolved.
I will likely want to revisit the architecture discussion, but that is for another day/version.

Please update this section to say "No open issues at this time."  (or some equivalent)